addons/binding/org.openhab.binding.plclogo/src/main/java/Moka7/IntByRef.java

```diff
@@ -0,0 +1,40 @@
+/*=============================================================================|
```

Comment:
Could you include the moka library as a jar, please also document the patch properly,